goes16-l2-cmi.json

```diff
@@ -0,0 +1,569 @@
+    "temporal_composition_schema": null,
+    "composition_function": null,
+    "collection_type": "collection",
+    "metadata": {
+        "providers": [
+            {
+                "url": "https://www.nasa.gov/content/goes",
+                "name": "National Aeronautics and Space Administration (NASA)",
+                "roles": [
+                    "producer"
+                ]
+            },
+            {
+                "url": "https://www.goes-r.gov/",
+                "name": "National Oceanic and Atmospheric Administration (NOAA)",
+                "roles": [
+                    "processor",
+                    "producer",
+                    "licensor"
+                ]
+            },
+            {
+                "url": "https://data.inpe.br/big/",
+                "name": "National Institute for Space Research (INPE)",
+                "roles": [
+                    "host",
+                    "processor"
+                ]
+            }
+        ],
+        "license": {
+            "type": "Public Domain",
+            "uri": "https://www.nesdisia.noaa.gov/policy.html"
+        }
+    },
+    "keywords": [
+        "goes",
+        "goes-16",
+        "level-2",
+        "cloud",
+        "moisture",
+        "meteorology",
+        "atmosphere",
+        "surface",
+        "brazil"
+    ],
+    "is_public": true,
+    "is_available": true,
+    "category": "eo",
+    "version": 1,
+    "version_predecessor": null,
+    "version_successor": null,
+    "extent": {
+        "spatial": {
+            "bbox": [
+            [
+              -180,
+              -81.33,
+              6.3,
+              81.33
+            ],
+            [
+              -116.0,
+              -56.0,
+              -25.0,
+              35.0
+            ]
+          ]
+        }
+    },
+    "summaries": {
+        "platform": [
+            "GOES-16"
+        ],
+        "constellation": [
+            "GOES-R Series"
+        ],
+        "instruments": [
+            "ABI"
+        ]
+    },
+    "item_assets": {
```

Comment:
Does this dataset have 33 assets (33 netcdf files) or does it have 1 asset (1 netcdf file, containing multiband)? Are there any other assets that should be provided alongise the .nc files? For instance .cdl?

Reply:
There are 33 different netcdf files:
- 16 files (one for each spectral band) in the original image projection (i.e. Geostationary Satellite View)
**assets**: `CMIP-C**-fdsk-nc`; _Note: There is a multiband version, but currently, we do not store it;_
- 1 file (band 02) in the original data acquisition projection but degraded from 500m to 1km resolution 
**asset**: `CMIP-C02-1km-fdsk-nc`;
- 16 files (one for each spectral band) of data remapped to EPSG:4326 and cropped to the South American region
**assets**: `CMIP-C**-remap-nc`.

We don't usually work with any other auxiliary files, such as .cdl, in the case of GOES-16. The information in the netCDF header/variables has been sufficient to describe and manipulate the data.